src/redprl/refiner.fun

```diff
@@ -17,7 +17,7 @@ struct
 
   structure Hyp =
   struct
-    fun Project z alpha jdg =
+    fun Project z _ jdg =
```

Comment:
The problem I was trying to solve is that the log is too long. How about I do what I did previously---writing `_(*alpha*)`?